### ISSUE TITLE
Training graph

### ISSUE THREAD

**Issue by jellyho** [microsoft/CogACT#24]
Hi, Thank you for your great work!

Can you share the pretraining loss graph?
I'm trying to test different VLM backbones and diffusion head,
so I want to see what is the reasonable graph when the model learned well.

**Comment by arnoldland** [comment]
Thanks for your interest! Here is the training loss of CogACT-Base. The training loss will initially decrease rapidly and then decrease more slowly over a long period of time.

![Image](https://github.com/user-attachments/assets/b3ed16e0-bab0-492e-a8fe-364ce1ba5f6d)

**Comment by jellyho** [comment]
Thank you for quick reply!
